packages/types/src/sessionVerification.ts

```diff
@@ -4,6 +4,7 @@ import type { SessionResource } from './session';
 import type { VerificationResource } from './verification';
 
 export interface __experimental_SessionVerificationResource extends ClerkResource {
+  // TODO: Is null possible ?
```

Comment:
@chanioxaris can u verify if having status == null is possible ?

Reply:
No status can't be null